examples/ads.amp.html

```diff
@@ -970,6 +971,16 @@ <h2 id="plista">Plista responsive widget</h2>
     <div placeholder></div>
     <div fallback></div>
   </amp-embed>
+  
+  <h2 id="polymorphicads">polymorphicAds</h2>
+  <amp-ad width=320 height=50
+      type="polymorphicads"
+      data-adunit="7c0b3ae742beccf94f7726ea832277a2"
+      data-params='{"testMode": "true"}'
+      >
```

Comment:
Thank you for review.

Closing bracket moved to previous line in new commit.
Also added renderStartImplemented in _config.js.